packages/plugin-workspace-tools/sources/commands/focus.ts

```diff
@@ -86,6 +86,7 @@ export default class WorkspacesFocus extends BaseCommand {
           workspace.manifest.devDependencies.clear();
         }
       } else {
+        workspace.manifest.setRawField(`_evictedFromFocusedInstall`, true);
```

Comment:
Why do you need a new `_evictedFromFocusedInstall`, rather than set `installConfig.selfReferences: false`?

Reply:
Because `installConfig.selfReferences` might be viewed as nm linker-specific code. I'm okay with either way.

Reply:
I'd prefer to avoid private fields like this as much as we can. Since we already need `selfReferences` anyway we should use it 👍

Reply:
Done